ctapipe/image/extractor.py

```diff
@@ -33,8 +33,8 @@
 
 @guvectorize(
     [
-        (float64[:], int64, int64, int64, float64, float64[:], float64[:]),
-        (float32[:], int64, int64, int64, float64, float64[:], float64[:]),
+        (float64[:], int64, int64, int64, float64, float32[:], float32[:]),
+        (float32[:], int64, int64, int64, float64, float32[:], float32[:]),
```

Comment:
Are you sure this actually changes the output dtype? Isn't it inferred and this just precompiles these two signatures? Not sure.

Reply:
I'm not sure - I would have thought if they are specified, it uses only them, but it may also infer... I'll add a test

Reply:
seems to work in the test at least

Reply:
Ok, thatn probably good.